### ISSUE TITLE
GraphQL Integration

### ISSUE THREAD

**Issue by shogunpurple** [Budibase/budibase#1405]

**Comment by shogunpurple** [comment]
nope

**Comment by Rsalyme** [comment]
Are you guys still interested in adding in this feature?

**Comment by shogunpurple** [comment]
@Rsalyme Yes but we have a lot of other higher priority items on our immediate roadmap, so we aren’t sure when we will get round to building it

**Comment by paul-asvb** [comment]
Thanks for the update. Do you guys have a public roadmap?

**Comment by dwstevens** [comment]
Being able to connect to Hygraph or similar graphql based headless CMS would be super helpful.

**Comment by tepene** [comment]
Roadmap 2022 has been closed. Is it still planned to integrate GraphQl?

**Comment by edmundo096** [comment]
> [mike12345567](https://github.com/mike12345567) added the [enhancement](https://github.com/Budibase/budibase/labels/enhancement) label [3 weeks ago](https://github.com/Budibase/budibase/issues/1405#event-13051317440)

... _3 weeks ago_ 👀

**Comment by dedfriedly-inspirdev** [comment]
Just wanted to see if this was still on the radar, have an data source just popped up graphql only so of course I'm more interested in this now

**Comment by mike12345567** [comment]
For anyone looking at this issue, I'd be interested to hear what people's thoughts are on how Budibase could integrate with a GraphQL endpoint - today it is achievable using the REST connector in Budibase (you can simply setup whatever queries you need) but if we were to implement a more complete GraphQL integration what would be expected of it.

If anyone has any thoughts on how this should look in Budibase would love to hear them as we'd want this to be more useful than the current ability to perform simple queries (for example, should it try to utilise introspection, etc).

**Comment by dedfriedly-inspirdev** [comment]
Hey @mike12345567 there's a few old discussions on this #1402 #2726 are a few that jump out to me if those are meaningful.

The schema introspection and type helping would be beyond amazing, obv making the data response tightly integrated for storage (which feels silly to even mention) and authentication/auth (again, silly to mention)
